sklearn/linear_model/sag.py

```diff
@@ -205,20 +205,14 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1., beta=0.,
     >>> X = np.random.randn(n_samples, n_features)
     >>> y = np.random.randn(n_samples)
     >>> clf = linear_model.Ridge(solver='sag')
-    >>> clf.fit(X, y)
-    ... #doctest: +NORMALIZE_WHITESPACE
+    >>> clf.fit(X, y) #doctest: +NORMALIZE_WHITESPACE
     Ridge(alpha=1.0, copy_X=True, fit_intercept=True, max_iter=None,
-          normalize=False, random_state=None, solver='sag', tol=0.001)
-
+       normalize=False, random_state=None, solver='sag', tol=0.001)
```

Comment:
Why did the indentation here change?

Reply:
My bad. Probably wrong key-stroke or something. Fixed.